### PR TITLE
Version 0.7

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#86 
#### What does this implement/fix? Explain your changes.
Release PR of Neighbors module's support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ Give a ⭐️ if this project helped you!
 ### Donate to our project
 If you do like our project and we hope that you do, can you please support us? Our project is not and is never going to be working for profit. We need the money just so we can continue doing what we do ;-) .			
 
-<a href="https://openscilab.com/#donation" target="_blank"><img src="https://github.com/openscilab/pymilo/raw/main/otherfiles/donation.png" height="90px" width="270px" alt="PyMilo Donation"></a>
+<a href="https://openscilab.com/#donation" target="_blank"><img src="https://github.com/openscilab/pymilo/raw/main/otherfiles/donation.png" height="90px" width="270px" alt="PyMilo Donation"></a>
```
Comment (sadrasabouri) [inline README.md:163]: I couldn't catch the change. Please make sure there is no unwanted hidden character insertion.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AHReccese) [reply README.md:163]: don't worry, nothing is changed.
my local git got confused according to some git stuff I was doing ...
@sadrasabouri

Comment (sadrasabouri) [reply README.md:163]: That's interesting :sweat_smile:

**Review by sadrasabouri — APPROVED** [review]

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**